### DISCUSSION TITLE
Error running multiple PVs with the API v3

### DISCUSSION THREAD

**Discussion by accioltd** [NREL/REopt-API-Analysis#121] [8 - API: Application Programming Interface]
I am trying to run multiple PVs, following the example provided in the multiple_pvs.json, but I get the following error:  "Unexpected Error. Please visit https://github.com/NREL/REopt-API-Analysis/discussions for support with common API usage issues. You can also email reopt@nrel.gov with your run_uuid (8502e0c3-f18c-4b56-ab4b-d42fba37f647) for support."
Can you help?
JSON source: https://github.com/NREL/REopt.jl/blob/master/test/scenarios/multiple_pvs.json

**Comment by Bill-Becker** [comment]
Sorry for the long delay on this request. We should have worked on this sooner. This is fixed now.

**Reply by accioltd** [reply]
Thanks a lot, Bill! I will test and start using next week.
